packages/babel-plugin-proposal-private-methods/test/fixtures/private-method-loose/reassignment/output.js

```diff
@@ -0,0 +1,17 @@
+var counter = 0;
+
+var _privateMethod = babelHelpers.classPrivateFieldLooseKey("privateMethod");
+
+class Foo {
+  constructor() {
+    Object.defineProperty(this, _privateMethod, {
+      value: _privateMethod2
+    });
+    babelHelpers.classPrivateFieldLooseBase(this, _privateMethod)[_privateMethod] = ++counter;
```

Comment:
Per https://github.com/tc39/ecma262/issues/1224, which got consensus at this meeting, I believe this is still incorrect. Even if `this` isn't an instance of `Foo`, we need to evaluate `++counter`:

1. Evaluate object
2. Evaluate key (which is a noop here, only computed props can have a evaluated key)
3. Evaluate RHS
4. Perform set

Reply:
Well, this is loose mode :shrug:

Reply:
Ha, I didn't see the `loose`. 👍

Reply:
Has it been covered in assumptions?